bpf/bpf_host.c

```diff
@@ -902,12 +902,22 @@ handle_netdev(struct __ctx_buff *ctx, const bool from_host)
 	return do_netdev(ctx, proto, from_host);
 }
 
+/*
+ * from-netdev is attached as a tc ingress filter to one or more physical devices
+ * managed by Cilium (e.g., eth0). This program is only attached when:
+ * - the host firewall is enabled, or
+ * - BPF NodePort is enabled
+ */
 __section("from-netdev")
 int from_netdev(struct __ctx_buff *ctx)
 {
 	return handle_netdev(ctx, false);
 }
 
+/*
+ * from-host is attached as a tc egress filter to the node's 'cilium_host'
+ * interface, or to the Flannel master device if present.
```

Comment:
s/`Flannel master`/`Flannel primary`

Reply:
Couldn't find any reference to neither `master` nor `primary` interfaces in the Flannel docs, it's all just 'interface'. I called it 'primary Flannel interface' instead, since the docs have some `ip a` output that contains an interface called `flannel.1` . :) Thanks!